.github/workflows/test_and_deploy.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     - name: Lint code base
       run: ruff check
     - name: Check for trailing whitespace and newlines at the file ends
-      run: ruff check --preview --select W291,W292,W391 *
+      run: ruff check --preview --select W291,W292,W391 
```

Comment:
Originally I copied it from a WIP version of Biotite's `next-major-release` branch. Later I found that it does not actually work as expected, so I completely removed it again. I think we can do this here as well.

In your current version it would run these selected rules on Python files, which is done anyway. Originally the idea was checking, all kinds of files (not only Python) for trailing whitespaces and line breaks.

```suggestion
```